src/test/java/org/traccar/TestIdentityManager.java

```diff
@@ -29,6 +29,14 @@ public Device getByUniqueId(String uniqueId) {
         return createDevice();
     }
 
+    public String getDevicePassword(long id) {
```

Comment:
You still have this method here. That's why you should annotate methods as `@Override`.

Reply:
Oh, you are right. Sorry I missed that. Thanks.